libs/go/libp2p_node/dht/dhtpeer/dhtpeer_test.go

```diff
@@ -1937,7 +1936,7 @@ func expectEnvelopeOrdered(t *testing.T, rx chan *aea.Envelope, counter int) {
 		}
 		message, _ := strconv.Atoi(string(envel.Message))
 		if message != counter {
-			log.Fatal(fmt.Sprintf("Expected counter %d received counter %d", counter, message))
+			log.Fatalf("Expected counter %d received counter %d", counter, message)
 		}
 	case <-timeout:
 		t.Error("Failed to receive envelope before timeout")
```

Comment:
Was this missed in the previous Go Lang PR?

Reply:
no, it was included